src/routes/connect/+page.svelte

```diff
@@ -298,6 +315,37 @@
         </div>
       {/if}
 
+      <div class="w-full">
+        <p class="text-neutral-600 dark:text-neutral-300">
+          {$translate('app.inputs.add_rune.recipes')}
+        </p>
+        <div class="flex justify-between">
+          {#each recipes as recipe, i}
```

Comment:
```suggestion
          {#each recipes as recipe}
```
We can remove the unused index var.